internal/pkg/source/source.go

```diff
@@ -462,6 +465,52 @@ func isSourceType(c classifier, t types.Type) bool {
 	}
 }
 
+// A type "can point" if it is itself a pointer or pointer-like type, or it contains
+// a pointer or pointer-like type.
+func canPoint(t types.Type) bool {
```

Comment:
I think this is a good point. Done. In addition, I changed the function name to `canBeTaintedByCall`, because that's really what the client code is interested in.